NewEvaluation.html

```diff
@@ -82,25 +88,26 @@
 
 		<div class="user-textarea">
 			<h3>Should the candidate be hired?</h3>
-			<textarea  style= "width:1200px ; height: 200px;" placeholder="The type of project that is suitable for the candidate &#13;&#10;Is guidance requiredd for the candidate"></textarea>
+			<textarea placeholder="The type of project that is suitable for the candidate &#13;&#10;Is guidance requiredd for the candidate"></textarea>
 		</div>
 
 		<div class="user-textarea">
 			<h3>General Impression</h3>
-			<textarea  style= "width:1200px ; height: 200px;" placeholder="*required" required></textarea>
+			<textarea placeholder="*required" required></textarea>
 		</div>
 
 		<div class="user-textarea">
 			<h3>Workflow, Leadership &#38; Soft Skills</h3>
-			<textarea  style= "width:1200px ; height: 200px;" placeholder="Describe the enviroment in which the candidate works. &#13;&#10;Describe ani guidance or management experience."></textarea>
+			<textarea placeholder="Describe the enviroment in which the candidate works. &#13;&#10;Describe ani guidance or management experience."></textarea>
```

Comment:
Will take it out. Forgot about it. Used it in the pure HTML version in order to make the page look somewhat decent